doc/merkle.md

```diff
@@ -60,7 +58,7 @@ a sum-tree over the commitments of outputs, and the negatives of the commitments
 of inputs.
 
 Input references are hashes of old commitments. It is a consensus rule that
-there are never two identical unspent outputs.
+there never are two identical unspent outputs.
```

Comment:
These both feel kind of awkward to me
Maybe this should be reworked as "all unspent outputs must be unique".

Reply:
Yeah that's a good idea :-)